<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AwTopoWidgetSettings</class>
 <widget class="QDialog" name="AwTopoWidgetSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mapping</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/AnyWave.qrc">
    <normaloff>:/images/AnyWave_logo.jpg</normaloff>:/images/AnyWave_logo.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Colormap:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="AwComboCMap" name="comboColormap"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxContours">
            <property name="text">
             <string>Show Contour Lines</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string># lines:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBoxContours"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxSensors">
            <property name="text">
             <string>Show Sensors</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="checkBoxLabels">
            <property name="text">
             <string>Show Labels</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonApply">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AwComboCMap</class>
   <extends>QComboBox</extends>
   <header location="global">widget/AwComboCMap.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/AnyWave.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonApply</sender>
   <signal>clicked()</signal>
   <receiver>AwTopoWidgetSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>AwTopoWidgetSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxContours</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBoxContours</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxSensors</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxLabels</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
